recipes/openssl/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
You need to use `ALL` not `all`. One or two months ago I tried to rename to `all`, but for some reason, CCI didn't accept well.

Reply:
wait, is it working now?! Amazing! :clap:

Reply:
The conversion also worked fine on Windows.
Some years ago, I remembered problems when changing cases.
Probably git fixed it.